doc/doxygen/references.bib

```diff
@@ -1328,6 +1328,45 @@ @article{Benson2002
   url =       {https://doi.org/10.1023/a:1020533003783}
 }
 
+% ------------------------------------
+% Step 81
+% ------------------------------------
+
+@BOOK{Schwartz1972,
+  AUTHOR={M. Schwartz},
+  TITLE={Principles of Electrodynamics},
+  Year={1972},
+  PUBLISHER={McGraw-Hill Book Company},
+  ADDRESS={New York},
+  SERIES={International Series in Pure and Applied Physics},
+}
+
+@BOOK{Monk2003,
+  AUTHOR={P. Monk},
+  TITLE={Finite Element Methods for {Maxwell's} Equations},
+  YEAR={2003},
+  PUBLISHER={Oxford University Press},
+  SERIES={Numerical Mathematics and Scientific Computation},
+}
+
+@ARTICLE{Geim2004,
+  AUTHOR={K.S. Novoselov and A.K. Geim, S.V. Morozov and D. Jiang and Y. Zhang and S.V. Dubonos and I.V. Grigorieva and A.A. Firsov},
```

Comment:
```suggestion
  AUTHOR={K.S. Novoselov and A.K. Geim and S.V. Morozov and D. Jiang and Y. Zhang and S.V. Dubonos and I.V. Grigorieva and A.A. Firsov},
```
I think you need this, otherwise bibtex will render things in a funny way.